[Linux] TypeError: Cannot read property 'channelMetadataRenderer' of undefined

Hi,

For past few days the local API stopped working for some reason:
![image](https://user-images.githubusercontent.com/2163486/116224114-eec85880-a758-11eb-9878-233552f93360.png)


**Environment Information (please complete the following information):**
 - OS: Linux
 - OS Version: Solus 4.2
 - FreeTube version: FreeTube-0.12.0-nightly-741.AppImage
 - Installation Method: AppImage
 - Primary API Used: Local API

I've tried with clean profile ( deleting the `~/.config/FreeTube` folder ) to no avail. 

Any suggestions how to troubleshoot this ?
Tried with VPN via USA, because I'm EU based and saw some comments that the new youtube notice window might break things, but it does not make any difference. 

Kind regards,
Presian

switched to linux channel

renderer.js:2 Uncaught TypeError: Cannot read property 'videoWidth' of null
    at o.checkAspectRatio (renderer.js:2)
    at renderer.js:2
renderer.js:2 
Array(24)
renderer.js:2 https://invidious.kavin.rocks
renderer.js:2 Error: Status code 404
    at ClientRequest.<anonymous> (/tmp/.mount_FreeTuNl…er/lib/parser.js:88)
    at Object.onceWrapper (events.js:422)
    at ClientRequest.emit (events.js:315)
    at HTTPParser.parserOnIncomingClient (_http_client.js:641)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:126)
    at HTTPParser.execute (<anonymous>)
    at TLSSocket.socketOnData (_http_client.js:509)
    at TLSSocket.emit (events.js:315)
    at addChunk (internal/streams/readable.js:309)
    at readableAddChunk (internal/streams/readable.js:284)
renderer.js:2 TypeError: Cannot read property 'channelMetadataRenderer' of undefined
    at YoutubeGrabberHelper.parseChannelVideoResponse (/tmp/.mount_FreeTuNl…fo/app/helper.js:60)
    at Function.getChannelVideosNewest (/tmp/.mount_FreeTuNl…chers/channel.js:10)
    at async Function.getChannelVideos (/tmp/.mount_FreeTuNl…tube-grabber.js:207)
/tmp/.mount_FreeTuNl…uter.common.js:2067 Uncaught (in promise) NavigationDuplicated: Avoided redundant navigation to current location: "/subscriptions".
    at createRouterError (/tmp/.mount_FreeTuNlblfG/resources/app.asar/node_modules/vue-router/dist/vue-router.common.js:2067:15)
    at createNavigationDuplicatedError (/tmp/.mount_FreeTuNlblfG/resources/app.asar/node_modules/vue-router/dist/vue-router.common.js:2037:15)
    at HashHistory.confirmTransition (/tmp/.mount_FreeTuNlblfG/resources/app.asar/node_modules/vue-router/dist/vue-router.common.js:2330:18)
    at HashHistory.transitionTo (/tmp/.mount_FreeTuNlblfG/resources/app.asar/node_modules/vue-router/dist/vue-router.common.js:2262:8)
    at HashHistory.push (/tmp/.mount_FreeTuNlblfG/resources/app.asar/node_modules/vue-router/dist/vue-router.common.js:2706:10)
    at /tmp/.mount_FreeTuNlblfG/resources/app.asar/node_modules/vue-router/dist/vue-router.common.js:3022:22
    at new Promise (<anonymous>)
    at VueRouter.push (/tmp/.mount_FreeTuNlblfG/resources/app.asar/node_modules/vue-router/dist/vue-router.common.js:3021:12)
    at o.navigate (file:///tmp/.mount_FreeTuNlblfG/resources/app.asar/dist/renderer.js:2:492430)
    at click (file:///tmp/.mount_FreeTuNlblfG/resources/app.asar/dist/renderer.js:2:492777)
renderer.js:2 
Array(24)
renderer.js:2 https://invidious.048596.xyz
renderer.js:2 Error: Status code 404
    at ClientRequest.<anonymous> (/tmp/.mount_FreeTuNl…er/lib/parser.js:88)
    at Object.onceWrapper (events.js:422)
    at ClientRequest.emit (events.js:315)
    at HTTPParser.parserOnIncomingClient (_http_client.js:641)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:126)
    at HTTPParser.execute (<anonymous>)
    at TLSSocket.socketOnData (_http_client.js:509)
    at TLSSocket.emit (events.js:315)
    at addChunk (internal/streams/readable.js:309)
    at readableAddChunk (internal/streams/readable.js:284)
renderer.js:2 TypeError: Cannot read property 'channelMetadataRenderer' of undefined
    at YoutubeGrabberHelper.parseChannelVideoResponse (/tmp/.mount_FreeTuNl…fo/app/helper.js:60)
    at Function.getChannelVideosNewest (/tmp/.mount_FreeTuNl…chers/channel.js:10)
    at async Function.getChannelVideos (/tmp/.mount_FreeTuNl…tube-grabber.js:207)
renderer.js:2 No sources available
renderer.js:2 Getting video info please wait...
renderer.js:2 Error: Status code: 429
    at ClientRequest.<anonymous> (/tmp/.mount_FreeTuNl…t/dist/index.js:204)
    at Object.onceWrapper (events.js:422)
    at ClientRequest.emit (events.js:315)
    at HTTPParser.parserOnIncomingClient (_http_client.js:641)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:126)
    at HTTPParser.execute (<anonymous>)
    at TLSSocket.socketOnData (_http_client.js:509)
    at TLSSocket.emit (events.js:315)
    at addChunk (internal/streams/readable.js:309)
    at readableAddChunk (internal/streams/readable.js:284)
invidious.048596.xyz…deos/M7RVPFisSO4?:1 Failed to load resource: the server responded with a status of 403 (Forbidden)
renderer.js:2 
Object
renderer.js:2 error
renderer.js:2 https://invidious.048596.xyz/api/v1/videos/M7RVPFisSO4?
renderer.js:2 Forbidden
renderer.js:2 found an error
renderer.js:2 
Object
renderer.js:2 
Object
renderer.js:2 -1
/tmp/.mount_FreeTuNl…/video.cjs.js:27343 Uncaught TypeError: Cannot read property 'length' of undefined
    at addPropertiesToMaster (/tmp/.mount_FreeTuNl…/video.cjs.js:27343)
    at parseMasterXml (/tmp/.mount_FreeTuNl…/video.cjs.js:29571)
    at DashPlaylistLoader.haveMaster_ (/tmp/.mount_FreeTuNl…/video.cjs.js:30148)
    at DashPlaylistLoader.onClientServerClockSync_ (/tmp/.mount_FreeTuNl…/video.cjs.js:30203)
    at DashPlaylistLoader.syncClientServerClock_ (/tmp/.mount_FreeTuNl…/video.cjs.js:30098)
    at /tmp/.mount_FreeTuNl…/video.cjs.js:30078
    at callbackWrapper (/tmp/.mount_FreeTuNl…/video.cjs.js:28882)
    at Object.callback (/tmp/.mount_FreeTuNl…/video.cjs.js:28904)
    at cbOnce (/tmp/.mount_FreeTuNl…ojs/xhr/index.js:95)
    at XMLHttpRequest.loadFunc (/tmp/.mount_FreeTuNl…js/xhr/index.js:162)
invidious.048596.xyz…PFisSO4?height=90:1 Failed to load resource: the server responded with a status of 404 (Not Found)
renderer.js:2 Uncaught TypeError: Cannot read property 'qualityLevels' of null
    at renderer.js:2
renderer.js:2 Uncaught TypeError: Cannot read property 'videoWidth' of null
    at o.checkAspectRatio (renderer.js:2)
    at renderer.js:2
renderer.js:2 Error: Request timed out after 60000ms
    at /tmp/.mount_FreeTuNl…r/lib/parser.js:101
        
america's funniest home videos channel
    
    No sources available
renderer.js:2 Getting video info please wait...
renderer.js:2 Error: Status code: 429
    at ClientRequest.<anonymous> (/tmp/.mount_FreeTuNl…t/dist/index.js:204)
    at Object.onceWrapper (events.js:422)
    at ClientRequest.emit (events.js:315)
    at HTTPParser.parserOnIncomingClient (_http_client.js:641)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:126)
    at HTTPParser.execute (<anonymous>)
    at TLSSocket.socketOnData (_http_client.js:509)
    at TLSSocket.emit (events.js:315)
    at addChunk (internal/streams/readable.js:309)
    at readableAddChunk (internal/streams/readable.js:284)
/tmp/.mount_FreeTuNl…ist/jquery.js:10109 GET https://invidious.048596.xyz/api/v1/videos/NXHyNQGNBIc? 403 (Forbidden)
renderer.js:2 
{readyState: 4, getResponseHeader: ƒ, getAllResponseHeaders: ƒ, setRequestHeader: ƒ, overrideMimeType: ƒ, …}
renderer.js:2 error
renderer.js:2 https://invidious.048596.xyz/api/v1/videos/NXHyNQGNBIc?
renderer.js:2 Forbidden
renderer.js:2 found an error
renderer.js:2 
{readyState: 4, getResponseHeader: ƒ, getAllResponseHeaders: ƒ, setRequestHeader: ƒ, overrideMimeType: ƒ, …}
renderer.js:2 
{readyState: 4, getResponseHeader: ƒ, getAllResponseHeaders: ƒ, setRequestHeader: ƒ, overrideMimeType: ƒ, …}
renderer.js:2 -1

painting the light channel

No sources available
renderer.js:2 Getting video info please wait...
renderer.js:2 Error: Status code: 429
    at ClientRequest.<anonymous> (/tmp/.mount_FreeTuNl…t/dist/index.js:204)
    at Object.onceWrapper (events.js:422)
    at ClientRequest.emit (events.js:315)
    at HTTPParser.parserOnIncomingClient (_http_client.js:641)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:126)
    at HTTPParser.execute (<anonymous>)
    at TLSSocket.socketOnData (_http_client.js:509)
    at TLSSocket.emit (events.js:315)
    at addChunk (internal/streams/readable.js:309)
    at readableAddChunk (internal/streams/readable.js:284)
/tmp/.mount_FreeTuNl…ist/jquery.js:10109 GET https://invidious.048596.xyz/api/v1/videos/S_cYlgl28Wo? 403 (Forbidden)
renderer.js:2 
{readyState: 4, getResponseHeader: ƒ, getAllResponseHeaders: ƒ, setRequestHeader: ƒ, overrideMimeType: ƒ, …}
renderer.js:2 error
renderer.js:2 https://invidious.048596.xyz/api/v1/videos/S_cYlgl28Wo?
renderer.js:2 Forbidden
renderer.js:2 found an error
renderer.js:2 
{readyState: 4, getResponseHeader: ƒ, getAllResponseHeaders: ƒ, setRequestHeader: ƒ, overrideMimeType: ƒ, …}
renderer.js:2 
{readyState: 4, getResponseHeader: ƒ, getAllResponseHeaders: ƒ, setRequestHeader: ƒ, overrideMimeType: ƒ, …}
renderer.js:2 -1
﻿
trending ... 
/tmp/.mount_FreeTuNl…ist/jquery.js:10109 GET https://invidious.048596.xyz/api/v1/trending/?region=US net::ERR_CONNECTION_RESET

UK, using UK IP

switching to Swiss IP ... 

turns out STL vid was premiere-bookmarked
america's funniest works, and comments too
painting the light works, and comments too
trending works

--- a/src/renderer/store/modules/subscriptions.js
+++ b/src/renderer/store/modules/subscriptions.js
@@ -4,7 +4,8 @@ const state = {
   allSubscriptionsList: [],
   profileSubscriptions: {
     activeProfile: MAIN_PROFILE_ID,
-    videoList: []
+    videoList: [],
+    errorChannels: []
   }
 }
 

--- a/src/renderer/views/Channel/Channel.js
+++ b/src/renderer/views/Channel/Channel.js
@@ -92,14 +92,9 @@ export default Vue.extend({
     },
 
     subscriptionInfo: function () {
-      const subIndex = this.activeProfile.subscriptions.findIndex((channel) => {
+      return this.activeProfile.subscriptions.find((channel) => {
         return channel.id === this.id
-      })
-      if (subIndex !== -1) {
-        return this.activeProfile.subscriptions[subIndex]
-      } else {
-        return null
-      }
+      }) ?? null
     },
 
     isSubscribed: function () {

--- a/src/renderer/views/Channel/Channel.js
+++ b/src/renderer/views/Channel/Channel.js
@@ -403,6 +403,7 @@ export default Vue.extend({
           this.bannerUrl = null
         }
 
+        this.errorMessage = ''
         this.isLoading = false
       }).catch((err) => {
         this.setErrorMessage(err.responseJSON.error)

--- a/src/renderer/views/Channel/Channel.js
+++ b/src/renderer/views/Channel/Channel.js
@@ -650,6 +650,7 @@ export default Vue.extend({
     },
 
     setErrorMessage: function (errorMessage) {
+      this.isLoading = false
       this.errorMessage = errorMessage
       this.id = this.subscriptionInfo.id
       this.channelName = this.subscriptionInfo.name

--- a/src/renderer/views/Channel/Channel.js
+++ b/src/renderer/views/Channel/Channel.js
@@ -50,6 +50,7 @@ export default Vue.extend({
       searchResults: [],
       shownElementList: [],
       apiUsed: '',
+      errorMessage: '',
       videoSelectValues: [
         'newest',
         'oldest',
@@ -90,16 +91,19 @@ export default Vue.extend({
       return this.$store.getters.getActiveProfile
     },
 
-    isSubscribed: function () {
+    subscriptionInfo: function () {
       const subIndex = this.activeProfile.subscriptions.findIndex((channel) => {
         return channel.id === this.id
       })
-
-      if (subIndex === -1) {
-        return false
+      if (subIndex !== -1) {
+        return this.activeProfile.subscriptions[subIndex]
       } else {
-        return true
+        return null
       }
+    },
+
+    isSubscribed: function () {
+      return this.subscriptionInfo !== null
     },
 
     subscribedText: function () {
@@ -251,6 +255,11 @@ export default Vue.extend({
       this.apiUsed = 'local'
       const expectedId = this.id
       ytch.getChannelInfo({ channelId: expectedId }).then((response) => {
+        if (response.alertMessage) {
+          this.setErrorMessage(response.alertMessage)
+          return
+        }
+        this.errorMessage = ''
         if (expectedId !== this.id) {
           return
         }
@@ -396,6 +405,7 @@ export default Vue.extend({
 
         this.isLoading = false
       }).catch((err) => {
+        this.setErrorMessage(err.responseJSON.error)
         console.log(err)
         const errorMessage = this.$t('Invidious API Error (Click to copy)')
         this.showToast({
@@ -636,6 +646,15 @@ export default Vue.extend({
           }
         }
       }
+    },
+
+    setErrorMessage: function (errorMessage) {
+      this.errorMessage = errorMessage
+      this.id = this.subscriptionInfo.id
+      this.channelName = this.subscriptionInfo.name
+      this.thumbnailUrl = this.subscriptionInfo.thumbnail
+      this.bannerUrl = null
+      this.subCount = null
     },
 
     handleFetchMore: function () {

--- a/src/renderer/views/Channel/Channel.vue
+++ b/src/renderer/views/Channel/Channel.vue
@@ -3,7 +3,7 @@
     ref="search"
   >
     <ft-loader
-      v-if="isLoading"
+      v-if="isLoading && !isErrorMessage"
       :fullscreen="true"
     />
     <ft-card

--- a/src/renderer/views/Channel/Channel.vue
+++ b/src/renderer/views/Channel/Channel.vue
@@ -3,7 +3,7 @@
     ref="search"
   >
     <ft-loader
-      v-if="isLoading && !isErrorMessage"
+      v-if="isLoading && !errorMessage"
       :fullscreen="true"
     />
     <ft-card

--- a/src/renderer/views/Channel/Channel.vue
+++ b/src/renderer/views/Channel/Channel.vue
@@ -61,6 +61,7 @@
         </div>
 
         <ft-flex-box
+          v-if="!errorMessage"
           class="channelInfoTabs"
         >
           <div
@@ -112,7 +113,7 @@
       </div>
     </ft-card>
     <ft-card
-      v-if="!isLoading"
+      v-if="!isLoading && !errorMessage"
       class="card"
     >
       <div
@@ -193,6 +194,14 @@
           <font-awesome-icon icon="search" /> {{ $t("Search Filters.Fetch more results") }}
         </div>
       </div>
+    </ft-card>
+    <ft-card
+      v-if="errorMessage"
+      class="card"
+    >
+      <p>
+        {{ errorMessage }}
+      </p>
     </ft-card>
   </div>
 </template>

--- a/src/renderer/views/Subscriptions/Subscriptions.css
+++ b/src/renderer/views/Subscriptions/Subscriptions.css
@@ -14,6 +14,10 @@
   right: 10px;
 }
 
+.channelBubble {
+  display: inline-block;
+}
+
 @media only screen and (max-width: 350px) {
   .floatingTopButton {
     position: absolute

--- a/src/renderer/views/Subscriptions/Subscriptions.js
+++ b/src/renderer/views/Subscriptions/Subscriptions.js
@@ -113,6 +113,7 @@ export default Vue.extend({
           }))
         } else {
           this.videoList = subscriptionList.videoList
+          this.errorChannels = subscriptionList.errorChannels
         }
       } else {
         this.getProfileSubscriptions()

--- a/src/renderer/views/Subscriptions/Subscriptions.js
+++ b/src/renderer/views/Subscriptions/Subscriptions.js
@@ -6,6 +6,7 @@ import FtButton from '../../components/ft-button/ft-button.vue'
 import FtIconButton from '../../components/ft-icon-button/ft-icon-button.vue'
 import FtFlexBox from '../../components/ft-flex-box/ft-flex-box.vue'
 import FtElementList from '../../components/ft-element-list/ft-element-list.vue'
+import FtChannelBubble from '../../components/ft-channel-bubble/ft-channel-bubble.vue'
 
 import ytch from 'yt-channel-info'
 import Parser from 'rss-parser'
@@ -19,13 +20,15 @@ export default Vue.extend({
     'ft-button': FtButton,
     'ft-icon-button': FtIconButton,
     'ft-flex-box': FtFlexBox,
-    'ft-element-list': FtElementList
+    'ft-element-list': FtElementList,
+    'ft-channel-bubble': FtChannelBubble
   },
   data: function () {
     return {
       isLoading: false,
       dataLimit: 100,
-      videoList: []
+      videoList: [],
+      errorChannels: []
     }
   },
   computed: {
@@ -123,6 +126,10 @@ export default Vue.extend({
     }
   },
   methods: {
+    goToChannel: function (id) {
+      this.$router.push({ path: `/channel/${id}` })
+    },
+
     getSubscriptions: function () {
       if (this.activeSubscriptionList.length === 0) {
         this.isLoading = false
@@ -144,10 +151,9 @@ export default Vue.extend({
 
       let videoList = []
       let channelCount = 0
-
+      this.errorChannels = []
       this.activeSubscriptionList.forEach(async (channel) => {
         let videos = []
-
         if (!this.usingElectron || this.backendPreference === 'invidious') {
           if (useRss) {
             videos = await this.getChannelVideosInvidiousRSS(channel)
@@ -174,7 +180,8 @@ export default Vue.extend({
 
           const profileSubscriptions = {
             activeProfile: this.activeProfile._id,
-            videoList: videoList
+            videoList: videoList,
+            errorChannels: this.errorChannels
           }
 
           this.videoList = await Promise.all(videoList.filter((video) => {
@@ -226,6 +233,11 @@ export default Vue.extend({
     getChannelVideosLocalScraper: function (channel, failedAttempts = 0) {
       return new Promise((resolve, reject) => {
         ytch.getChannelVideos({ channelId: channel.id, sortBy: 'latest' }).then(async (response) => {
+          if (response.alertMessage) {
+            this.errorChannels.push(channel)
+            resolve([])
+            return
+          }
           const videos = await Promise.all(response.items.map(async (video) => {
             if (video.liveNow) {
               video.publishedDate = new Date().getTime()
@@ -297,33 +309,38 @@ export default Vue.extend({
           resolve(items)
         }).catch((err) => {
           console.log(err)
-          const errorMessage = this.$t('Local API Error (Click to copy)')
-          this.showToast({
-            message: `${errorMessage}: ${err}`,
-            time: 10000,
-            action: () => {
-              navigator.clipboard.writeText(err)
-            }
-          })
-          switch (failedAttempts) {
-            case 0:
-              resolve(this.getChannelVideosLocalScraper(channel, failedAttempts + 1))
-              break
-            case 1:
-              if (this.backendFallback) {
-                this.showToast({
-                  message: this.$t('Falling back to Invidious API')
-                })
-                resolve(this.getChannelVideosInvidiousRSS(channel, failedAttempts + 1))
-              } else {
-                resolve([])
+          if (err.toString().match(/404/)) {
+            this.errorChannels.push(channel)
+            resolve([])
+          } else {
+            const errorMessage = this.$t('Local API Error (Click to copy)')
+            this.showToast({
+              message: `${errorMessage}: ${err}`,
+              time: 10000,
+              action: () => {
+                navigator.clipboard.writeText(err)
               }
-              break
-            case 2:
-              resolve(this.getChannelVideosLocalScraper(channel, failedAttempts + 1))
-              break
-            default:
-              resolve([])
+            })
+            switch (failedAttempts) {
+              case 0:
+                resolve(this.getChannelVideosLocalScraper(channel, failedAttempts + 1))
+                break
+              case 1:
+                if (this.backendFallback) {
+                  this.showToast({
+                    message: this.$t('Falling back to Invidious API')
+                  })
+                  resolve(this.getChannelVideosInvidiousRSS(channel, failedAttempts + 1))
+                } else {
+                  resolve([])
+                }
+                break
+              case 2:
+                resolve(this.getChannelVideosLocalScraper(channel, failedAttempts + 1))
+                break
+              default:
+                resolve([])
+            }
           }
         })
       })
@@ -403,25 +420,30 @@ export default Vue.extend({
               navigator.clipboard.writeText(err)
             }
           })
-          switch (failedAttempts) {
-            case 0:
-              resolve(this.getChannelVideosInvidiousScraper(channel, failedAttempts + 1))
-              break
-            case 1:
-              if (this.backendFallback) {
-                this.showToast({
-                  message: this.$t('Falling back to the local API')
-                })
-                resolve(this.getChannelVideosLocalRSS(channel, failedAttempts + 1))
-              } else {
+          if (err.toString().match(/500/)) {
+            this.errorChannels.push(channel)
+            resolve([])
+          } else {
+            switch (failedAttempts) {
+              case 0:
+                resolve(this.getChannelVideosInvidiousScraper(channel, failedAttempts + 1))
+                break
+              case 1:
+                if (this.backendFallback) {
+                  this.showToast({
+                    message: this.$t('Falling back to the local API')
+                  })
+                  resolve(this.getChannelVideosLocalRSS(channel, failedAttempts + 1))
+                } else {
+                  resolve([])
+                }
+                break
+              case 2:
+                resolve(this.getChannelVideosInvidiousScraper(channel, failedAttempts + 1))
+                break
+              default:
                 resolve([])
-              }
-              break
-            case 2:
-              resolve(this.getChannelVideosInvidiousScraper(channel, failedAttempts + 1))
-              break
-            default:
-              resolve([])
+            }
           }
         })
       })

--- a/src/renderer/views/Subscriptions/Subscriptions.vue
+++ b/src/renderer/views/Subscriptions/Subscriptions.vue
@@ -8,6 +8,22 @@
       v-else
       class="card"
     >
+      <div
+        v-if="errorChannels.length !== 0"
+      >
+        <h3> {{ $t("Subscriptions.Error Channels") }}</h3>
+        <div>
+          <ft-channel-bubble
+            v-for="(channel, index) in errorChannels"
+            :key="index"
+            :channel-name="channel.name"
+            :channel-id="channel.id"
+            :channel-thumbnail="channel.thumbnail"
+            class="channelBubble"
+            @click="goToChannel(channel.id)"
+          />
+        </div>
+      </div>
       <h3>{{ $t("Subscriptions.Subscriptions") }}</h3>
       <ft-flex-box
         v-if="activeVideoList.length === 0"

--- a/static/locales/en-US.yaml
+++ b/static/locales/en-US.yaml
@@ -79,7 +79,7 @@ Subscriptions:
     # On Subscriptions Page
   Subscriptions: Subscriptions
   # channels that were likely deleted
-  Error Channels: Error Channels
+  Error Channels: Channels with Errors
   Latest Subscriptions: Latest Subscriptions
   This profile has a large number of subscriptions.  Forcing RSS to avoid rate limiting: This
     profile has a large number of subscriptions.  Forcing RSS to avoid rate limiting

--- a/static/locales/en-US.yaml
+++ b/static/locales/en-US.yaml
@@ -78,6 +78,8 @@ Search Filters:
 Subscriptions:
     # On Subscriptions Page
   Subscriptions: Subscriptions
+  # channels that were likely deleted
+  Error Channels: Error Channels
   Latest Subscriptions: Latest Subscriptions
   This profile has a large number of subscriptions.  Forcing RSS to avoid rate limiting: This
     profile has a large number of subscriptions.  Forcing RSS to avoid rate limiting

